07-13-16-language-plugins.md

```diff
@@ -0,0 +1,94 @@
+## A Milestone for Tessel
+This week we merged one of our most significant pull requests: [adding Rust deployment support](https://github.com/tessel/t2-cli/pull/790). For the past three years we've focused exclusively on deploying JavaScript on microcontrollers but this is our first step into a new language and a new community. We're not leaving JavaScript behind, rather, we see a future where the languages compliment each other (more on that in a subsequent post).
```

Comment:
...on microcontrollers. This is our first step (in that time? something feels missing)